bigframes/version.py

```diff
@@ -12,8 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import datetime
+import warnings
+
 __version__ = "1.39.0"
 
 # {x-release-please-start-date}
 __release_date__ = "2025-03-05"
 # {x-release-please-end}
+
+if datetime.datetime.today() - datetime.datetime.strptime(
+    __release_date__, "%Y-%m-%d"
+) > datetime.timedelta(days=365):
+    msg = f"Your BigFrames version {__version__} is beyond the support period. Please update to the lastest version."
```

Comment:
This feels a bit too precise. Support policies can change. Let's just say it's greater than 1 year old.